StripePaymentsUI/StripePaymentsUI/Source/UI Components/STPFormView.swift

```diff
@@ -65,6 +65,22 @@ public class STPFormView: UIView, STPFormInputValidationObserver {
     static let cornerRadius: CGFloat = 6
     static let interSectionSpacing: CGFloat = 7
 
+    public override var intrinsicContentSize: CGSize {
```

Comment:
Any reason we can't just override this in the STPCardFormView.Representable? I think overriding this where we us constraints and intrinsicContentSize may not be correct but not sure.